crates/sui-replay/src/types.rs

```diff
@@ -21,7 +21,7 @@ use tracing::error;
 
 // These are very testnet specific
 pub(crate) const TESTNET_GENESIX_TX_DIGEST: &str = "Cgww1sn7XViCPSdDcAPmVcARueWuexJ8af8zD842Ff43";
-pub(crate) const SAFE_MODE_TX_1_DIGEST: &str = "AGBCaUGj4iGpGYyQvto9Bke1EwouY8LGMoTzzuPMx4and";
+pub(crate) const SAFE_MODE_TX_1_DIGEST: &str = "AGBCaUGj4iGpGYyQvto9Bke1EwouY8LGMoTzzuPMx4nd";
```

Comment:
Did you mean to change this?

Reply:
Yup! it was accidentally changed by @lxfind in a refactor PR.

Reply:
oops, my bad! spelling-check attack